src/js/components/sharedComponents/header/DropdownItem.jsx

```diff
@@ -39,9 +40,20 @@ const DropdownItem = ({
     appendToExistingUrl = false
 }) => {
     const { pathname, search } = useLocation();
-    const newUrl = appendToExistingUrl
-        ? getNewUrlForGlossary(pathname, url, search)
-        : url;
+
+    let newUrl = url;
+    if (appendToExistingUrl && pathname.includes('about-the-data')) {
+        newUrl = getNewUrlForATD(pathname, url, search);
+    } else if (appendToExistingUrl && pathname.includes('glossary')) {
```

Comment:
Something here broken the code for the Glossary.  Now when trying to open the Glossary from an internal page, you're redirected to the homepage.  The condition for ATD could removed here since you have the additional condition below to open the About the Data from the redux action